packages/graphql/tests/schema/issues/3439.test.ts

```diff
@@ -0,0 +1,3214 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { printSchemaWithDirectives } from "@graphql-tools/utils";
+import { gql } from "graphql-tag";
+import { lexicographicSortSchema } from "graphql/utilities";
+import { Neo4jGraphQL } from "../../../src";
+import { TestSubscriptionsPlugin } from "../../utils/TestSubscriptionPlugin";
+import { validateSchema } from "graphql";
+
+describe("https://github.com/neo4j/graphql/issues/3439", () => {
```

Comment:
Maybe we could add an integration/e2e test to replicate this issue?

Schema tests are good but a bit unreliable on their own

Reply:
The magic sauce of this schema test is that I added validation using `validateSchema` - this is an exact reproduction of this issue in that they weren't able to generate the schema, and these tests demonstrate that they now can.